definitions/variable/energy/energy-final.yaml

```diff
@@ -106,6 +106,18 @@
     energy_use: all excl. losses
     sector: Residential and Commercial
     unit: [EJ/yr, GWh]
+- Final Energy|Residential and Commercial|Space and Water Heating:
+    description: Final energy consumed for space and water heating in residential and commercial buildings
+    energy_source: all
+    energy_use: all excl. losses
+    sector: Residential and Commercial
+    unit: [EJ/yr, GWh]
```

Comment:
Do you need to allow two different variables?
They also seem to have two different dimensions, Energy/time vs. Energy.

Reply:
I kept both units to be consistent with related existing variables. But I go with `EJ/yr` now as for the time being I'll be the only one using it...